utils/debug/Debug.js

```diff
@@ -101,16 +102,17 @@ function Debug(view, datDebugTool, chartDivContainer) {
     initialPosition.crs = view.referenceCrs;
 
     const getCenter = (controls && controls.getCameraTargetPosition) ? controls.getCameraTargetPosition : () => view.camera.camera3D.position;
-    const cameraTargetListener = () => {
-        initialPosition.setFromVector3(getCenter()).as('EPSG:4326', geoPosition);
+    const cameraTargetListener = (event) => {
+        initialPosition.setFromVector3(
+            view.getPickingPositionFromDepth(view.eventToViewCoords(event)) || new THREE.Vector3(),
```

Comment:
could you replace `undefined` picking by `not Number` value like `-`?

Reply:
I changed it. Now the coordinates will display `---------` if undefined.